src/char/char_mapif.cpp

```diff
@@ -299,10 +299,11 @@ int chmapif_parse_askscdata(int fd){
 			{
 				Sql_GetData(sql_handle, 0, &data, NULL); scdata.type = atoi(data);
 				Sql_GetData(sql_handle, 1, &data, NULL); scdata.tick = strtoll( data, nullptr, 10 );
-				Sql_GetData(sql_handle, 2, &data, NULL); scdata.val1 = atoi(data);
-				Sql_GetData(sql_handle, 3, &data, NULL); scdata.val2 = atoi(data);
-				Sql_GetData(sql_handle, 4, &data, NULL); scdata.val3 = atoi(data);
-				Sql_GetData(sql_handle, 5, &data, NULL); scdata.val4 = atoi(data);
+				Sql_GetData(sql_handle, 2, &data, NULL); scdata.tick_total = strtoll(data, nullptr, 10);
+				Sql_GetData(sql_handle, 3, &data, NULL); scdata.val1 = atoi(data);
+				Sql_GetData(sql_handle, 4, &data, NULL); scdata.val2 = atoi(data);
+				Sql_GetData(sql_handle, 5, &data, NULL); scdata.val3 = atoi(data);
+				Sql_GetData(sql_handle, 6, &data, NULL); scdata.val4 = atoi(data);
```

Comment:
Move it to index 6 instead. It does not hurt us and will not break as much if someone has a customization.